python/cuml/ensemble/randomforest_common.pyx

```diff
@@ -347,7 +345,7 @@ class BaseRandomForestModel(Base):
                               fil_sparse_format, threshold=0.5,
                               output_class=False,
                               predict_proba=False) -> CumlArray:
-        _, n_rows, n_cols, dtype = \
+        _, _, _, _ = \
```

Comment:
Is this call to `input_to_cuml_array` doing anything? Its results are not used but it might be doing some error checking / validation?

Reply:
It will set the classe's input_dtype. Getting rid of unexpected side-effects like that is one of my refactoring goals.

Reply:
@wphicks Can you push changes or file an issue if you'd like to see a change here? I don't have enough familiarity with this code, unfortunately.